app/components/location_subject_filter_component.html.erb

```diff
@@ -13,6 +13,6 @@
         @results.filter_params_for(root_path),
         class: 'govuk-link--no-visited-state govuk-!-margin-left-4',
         data: { qa: 'link' }) do %>
-    Change <span class="govuk-visually-hidden"><%= @results.provider_filter? ? 'subject or provider' : 'subject or location' %></span>
+    Change <span class="govuk-visually-hidden">&nbsp;<%= @results.provider_filter? ? 'subject or provider' : 'subject or location' %></span>
```

Comment:
You only need to move the space from outside the span to inside the span.

```
Change<span class="govuk-visually-hidden"> <%= @results.provider_filter? ? 'subject or provider' : 'subject or location' %></span>
```

But having the `&nbsp;` within the span is also okay.

You can remove the space between 'change' and the span.

Reply:
Like this? https://github.com/DFE-Digital/find-teacher-training/pull/1608/commits/38ff0916f42ab0af0126ee85502011f03b843561